src/UpdateJulia.jl

```diff
@@ -402,7 +404,7 @@ function migrate_packages(v, force=false)
     else
         mkpath(dirname(target))
         cp(source, target, force=force)
-        run(`julia-$(v.major).$(v.minor) -e "using Pkg; Pkg.update()"`)
+        run(`julia-$(v.major).$(v.minor) -e "using Pkg; Pkg.activate(\"$(escape_string(dirname(target)))\"); Pkg.update()"`)
```

Comment:
It's bad to not respect a user's non-standard default environment, but we copy global environment using standard paths, so we should instantiate it using standard paths. A future version may support nonstandard environment locations—or not.

Reply:
If we assume that their nonstandard environment is not Julia version dependent, then there is nothing to do in that case.

Reply:
Documented in a4cf134